### ISSUE TITLE
Allow measurements to contain Value-castable objects

### ISSUE THREAD

**Issue by eliperelman** [gobwas/influent#3]
Sometimes the data I write to InfluxDB comes from a JSON file of influent data points which I store and write to database at a later time. Measurement `Value`s are JSON-ified, which become objects. Unfortunately you can't run these back through influent because measurement values are expected to not be objects.

I'd like to propose that objects can be passed back through `writeMany`, etc., like so:

``` js
client.writeMany([
  {
    fields: {
      example: {
        data: 123,
        type: 2
      }
    }
  }
]);
```

So basically if fields contains an object, we determine that it has `data` and optionally `type`, and inside influent, create a `new Value(data, type)`. If this is something we can consider, I have this change made locally and can submit a PR.

**Comment by gobwas** [comment]
This is an good idea in stream of `DecoratorClient`, so I've implemented it already. Is [this](https://github.com/gobwas/influent/pull/4/files#diff-48f34f38f2c6088fbecf9d8c02263d8bL22) what you want?

**Comment by eliperelman** [comment]
Yep! Specifically, this:

https://github.com/gobwas/influent/pull/4/files#diff-48f34f38f2c6088fbecf9d8c02263d8bR28

That should take care of that use case.

**Comment by gobwas** [comment]
Merged :ocean:
